jnosql-mapping/jnosql-jupiter/pom.xml

```diff
@@ -35,7 +35,6 @@
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-api</artifactId>
-            <version>${junit.version}</version>
```

Comment:
@dearrudam if you remove this version line ti won't work.

Reply:
@otaviojava this dependency is managed by the `jnosql-parent`'s POM... the `mvn -B package` command works as expected, we can see it by taking a look at the check results [here](https://github.com/eclipse/jnosql/pull/310/checks)

Reply:
ahhh ok, thanks :)